packages/migrations/src/actions/2024.07.16T13-44-00.oidc-only-access.ts

```diff
@@ -0,0 +1,9 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2024.07.16T13-44-00.oidc-only-access.ts',
+  run: ({ sql }) => sql`
+    ALTER TABLE "oidc_integrations"
+    ADD COLUMN "oidc_user_access_only" BOOLEAN NOT NULL DEFAULT TRUE;
```

Comment:
adding true by default break someone?

Reply:
we have very little rows and long term it was easier to just get a boolean and not do `?? true` here and there

Reply:
> will adding true by default break someone?
it won't affect existing accounts, only if somebody wants to now invite a new member.
I think it should be the default anyway, otherwise people can invite 3rd-party accounts and one goal of setting up SSO is to limit access anyway.

Reply:
```
 do ?? true
```

It would be 
```ts
const Model = zod.object({
  // ... other fields
  oidcUserAccessOnly: zod.boolean().nullable().transform((value) => value === null ? true : value)
})
```

Reply:
yeah but why should I do it in code, if I can do it at db level?

Reply:
and now we need to make sure that the model is correct and used everywhere we fetch oidc integration

Reply:
just to shave off 30ms